packages/@magic-sdk/types/src/modules/auth-types.ts

```diff
@@ -162,6 +183,17 @@ export type LoginWithMagicLinkEventHandlers = {
   [LoginWithMagicLinkEventEmit.Retry]: () => void;
 } & DeviceVerificationEventHandlers;
 
+export type LoginWithSmsOTPEventHandlers = {
+  // Event sent
+  [LoginWithSmsOTPEventEmit.VerifySmsOtp]: (otp: string) => void;
+  [LoginWithSmsOTPEventEmit.Cancel]: () => void;
```

Comment:
[LoginWithSmsOTPEventEmit.Retry]: () => void;

We need a retry event for whitelabel flow